api/envoy/config/rbac/v2alpha/rbac.proto

```diff
@@ -137,7 +142,17 @@ message Principal {
   message Authenticated {
     // The name of the principal. If set, the URI SAN is used from the certificate, otherwise the
     // subject field is used. If unset, it applies to any user that is authenticated.
-    string name = 1;
+    //
+    // .. attention::
+    //
+    //   This field is deprecated. Use `principal_name` field instead. If both `name` and
+    //   `principal_name` is set, only the `principal_name` will have effect.
+    //
+    string name = 1 [deprecated = true];
```

Comment:
We don't strictly need to deprecate for an API in `v2alpha` status. If you need to deprecate because you suspect there are existing users who need this, this is fine, otherwise I would just make the change without going through a deprecation cycle. Also,you can safely use `oneof` between new/old without breaking wire compat.

Reply:
Okay, then I'll just remove `name`, I suspect it's only used by Istio, should be easy to change to the new `principal_name` field.